packages/website/pages/docs/guides/enable-a-proposer.mdx

```diff
@@ -78,6 +78,11 @@ If you are not running a local prover, then you can specify a prover from the [p
 
 - `PROVER_ENDPOINTS`
 
+To select multiple prover endpoints at the same time
+```shell
+PROVER_ENDPOINTS=<PROVER_ENDPOINT_1>:<PROVER_ENDPOINT_2>:<PROVER_ENDPOINT_3>
```

Comment:
it is a comma, you can check in the comments of .env.sample of simple-taiko-node 

else lgtm 🙂